<div class="alert alert-block alert-info">

__6/6__ Ananya Sriram

# Donovan Feist

# CH EN 3603

# Homework 5

# Problem 2

Water is used to remove $CO_2$ from air by absorption in a column packed with Pall rings described in
Chapter 6. At a region of the column where the partial pressure of $CO_2$ at the interface is $150 psia$
and the concentration in the bulk liquid is negligible, the absorption rate is $0.017 \frac{lbmol}{h\cdot ft^2}$. The $CO_2$ diffusivity in water is $2.0 \times 10^{-5} \frac{cm2}{s}$. Henry’s law for $CO_2$ is $p = Hx$, where $H = 9,000 psia$.

Calculate:

(a) liquid-phase mass-transfer coefficient and film thickness;

(b) contact time for the penetration theory; and

(c) average eddy residence time and the probability distribution for the surface-renewal theory.

For part (c), just calculate s - the surface renewal rate.

## Solution

### part a

Because there is no bulk flow of $CO_2$,

$N_{CO_2}=J_{CO_2}=-cD_{AB} \nabla x_{CO_2}=-cD_{AB} \frac{d x_{CO_2}}{dz}$.

Using film theory this leads to

$N_{CO_2}=-c\frac{D_{AB}}{\delta}(x_{CO_{2_{fluid}}}-x_{CO_{2_{gas}}})$.

The concentration of water in the fluid is

$c=\frac{\rho}{MW_{water}}$

This leads to

$N_{CO_2}=(\frac{\rho_{water}}{MW_{water}})\frac{D_{AB}}{\delta}(x_{CO_{2_{gas}}}-x_{CO_{2_{fluid}}})$.

Let $k_c=\frac{D_{AB}}{\delta} \Rightarrow

$N_{CO_2}=(\frac{\rho_{water}}{MW_{water}})k_c(x_{CO_{2_{gas}}}-x_{CO_{2_{fluid}}})$.

Solving for $k_c \Rightarrow$

$k_c=\frac{N_{CO_2} MW_{water}}{\rho_{water} (x_{CO_{2_{gas}}}-x_{CO_{2_{fluid}}})}$

In [8]:
NCO2=0.017 #lbmol/(h ft2)
NCO2metric=NCO2*453.59237/(3600*12**2*2.54**2) #mol/(s cm2)
MWwater=18.01528 #g/mol
ρwater=1 #g/cm3
Dab=2*10**(-5) #cm2/s
p=150 #psia
H=9000 #psia
xCO2gas=p/H
xCO2fluid=0

kc=NCO2metric*MWwater/(ρwater*(xCO2gas-xCO2fluid))

print('The mass transfer coefficient = {0:1.4} cm/s.'.format(kc))

δ=Dab/kc

print('The film thickness = {0:1.4} cm.'.format(δ))

The mass transfer coefficient = 0.002492 cm/s.
The film thickness = 0.008025 cm.


$\boxed{k_c = 0.002492 cm/s}$

$\boxed{\delta = 0.008025 cm}$

### part b

Using the Higie penetration theory

$k_c=2\sqrt{\frac{D_{AB}}{\pi t_c}} \Rightarrow t_c=\frac{D_{AB}}{\pi (\frac{k_c}{2})^2}$

In [14]:
from numpy import pi

tc=Dab/(pi*(kc/2)**2)

print('The contact time = {0:1.5} s.'.format(tc))

The contact time = 4.1001 s.


$\boxed{t_c=4.1001 s}$

### part c

From the surface renewal model

$k_c=\sqrt{D_{AB} s} \Rightarrow s=\frac{k_c^2}{D_{AB}}$

In [15]:
s=kc**2/Dab

print(s)

0.3105410447215787


$\boxed{surface \space renewal \space time = s=0.3105 \frac{1}{s}}$